src/renderer/scss/component/_shapeshift.scss

```diff
@@ -0,0 +1,63 @@
+.shapeshift__wrapper.shapeshift__initial-wrapper {
+  min-height: 311px;
```

Comment:
This scares me.

Reply:
Yeah...

Can' think of another way to do it. Nothing bad could really happen, at least nothing worse than not having this. If someone changes the form content to be a different height, it will just "jump" to that height after it's rendered. The same thing would happen if we don't have this style, it will "jump" and increase the height when the form is rendered.

Happy to change it if we can think of a better way. I just hate having content jumping when stuff is loading

Reply:
I don't see a better way of doing this either. As you said, worst case scenario, it will just do what it otherwise would have done had you not put this in. Best case scenario, it looks great.

But yeah, absolute px values are never great for dimensions :P

Reply:
I'm okay shipping this as-is.

However, if we wanted to avoid it, I think there's a potential solution where we render the component before shapeshift is ready, but render it with a very low opacity or `visibility: hidden`. In addition, either turn off interaction or render a blocking element on top of entire component. Then we can absolutely center a loading icon on top of this, and have it match the height precisely.

Reply:
👍